test/test_nmt_bins.py

```diff
@@ -62,6 +62,13 @@ def test_bins_edges(self):
         self.assertTrue(np.sum(np.fabs(self.bc.get_effective_ells() -
                                        self.be.get_effective_ells())) < 1E-10)
 
+    def test_min_max(self):
+        n = self.get_n_bands()
```

Comment:
I think you want `self.be.get_n_bands()` here

Reply:
Thanks - I couldn't face compiling on my laptop and messing with my current install, so thought I'd leave it to travis to run the tests.